### PR TITLE
feat: Add customers listing api ref

### PR DESCRIPTION
**Pull request by jdenquin**

### CODE REVIEW

File: docs/api/05_subscriptions/create-subscription.mdx [hunk 1]
```diff
@@ -131,23 +131,21 @@ import TabItem from '@theme/TabItem';
...
 }
 ```
 
 | Attributes | Type | Description |
 | -----------| -----| ----------- |
-| customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
+| external_customer_id | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Customer unique identifier in your application |
 | plan_code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the plan.<br></br>It must match the `code` property of one of the plans. |
-| unique_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `customer_id`. |
+| external_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique external identifier of the subscription. Used as an idempotency key. Can also be used for upgrades/downgrades. Default value if not provided: `external_customer_id`. |
```
Comment (mathieu-lago) [inline docs/api/05_subscriptions/create-subscription.mdx:147]: I believe `external_id` is now required and the fallback has been removed (i.e. does not default to `external_customer_id` anymore).

File: docs/api/06_events/create-event.mdx [hunk 2]
```diff
@@ -161,8 +161,8 @@ api/v1/events
...
 | -----------| -----| ----------- |
 | **transaction_id** | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Unique ID identifying the event. <br></br>As it will be used for idempotency, it should be a unique identifier |
-| **subscription_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique subscription ID in Lago's application.<br></br>`subscription_id` is required if the customer has multiple subscriptions or if `customer_id` is not provided (in case there's only one subscription) |
-| **customer_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Customer unique identifier in your application.<br></br>If `subscription_id` is not given, `customer_id` is required if there is only one subscription attached to customer. For multiple subscriptions per customer this attribute is not enough |
+| **external_subscription_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Unique subscription ID in Lago's application.<br></br>`external_subscription_id` is required if the customer has multiple subscriptions or if `external_customer_id` is not provided (in case there's only one subscription) |
```
Comment (mathieu-lago) [inline docs/api/06_events/create-event.mdx:164]: Replace "Unique subscription ID in Lago's application." with "Unique subscription ID in your application."

File: docs/api/06_events/create-batch-event.mdx [hunk 3]
```diff
@@ -126,8 +126,8 @@ api/v1/events/batch
 | Attributes | Type | Description |
 | -----------| -----| ----------- |
 | **transaction_id** | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Unique ID identifying the event. <br></br>As it will be used for idempotency, it should be a unique identifier |
-| **subscription_ids** | Array &nbsp &nbsp &nbsp<Required>**Required**</Required> | Array of subscription ids that belongs to one customer |
-| **customer_id** | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Customer unique identifier in your application |
+| **external_subscription_ids** | Array &nbsp &nbsp &nbsp<Required>**Required**</Required> | Array of subscription ids that belongs to one customer |
```
Comment (mathieu-lago) [inline docs/api/06_events/create-batch-event.mdx:129]: "Array of subscription IDs associated with the customer."

File: docs/guide/09_events/ingesting_events.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 1
 ---
 
 # Ingesting Events
-This guide explains how Lago ingests usage-based events coming from you product. 
+This guide explains how Lago ingests usage-based events coming from you product.
```
Comment (mathieu-lago) [inline docs/guide/09_events/ingesting_events.md:6]: "coming from your application."

File: docs/api/06_events/create-event.mdx (hunk 2 above)
Comment (rsempe) [reply docs/api/06_events/create-event.mdx:164]: Thanks Mathieu - Updating the base branch about external id 👍

File: docs/api/05_subscriptions/create-subscription.mdx (hunk 1 above)
Comment (rsempe) [reply docs/api/05_subscriptions/create-subscription.mdx:147]: Thanks Mathieu - Updating the base branch about external id 👍

File: docs/api/06_events/create-batch-event.mdx (hunk 3 above)
Comment (rsempe) [reply docs/api/06_events/create-batch-event.mdx:129]: Thanks Mathieu - Updating the base branch about external id 👍

**Pull request merged**